### PR TITLE
Use issue form for support requests

### PR DESCRIPTION
**Pull request by sftim**
Use a GitHub issue form to tell folks that this GitHub repo isn't the right venue to ask for help with Kubernetes.

Relevant to https://github.com/kubernetes/community/issues/5972

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/support.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+labels: [kind/support]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **STOP** -- **PLEASE READ!**
+        GitHub is not the right place for support requests.
+
+        You can instead post your question on the [Kubernetes Slack](http://slack.k8s.io/) or the [Discuss Kubernetes](https://discuss.kubernetes.io/) forum.
+        You can also check on [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes)
```
Comment (kbhawkey) [inline .github/ISSUE_TEMPLATE/support.yaml:13]: nit: Add a period at end of sentence (line 13).

**Comment by kbhawkey** [comment]
@sftim , As @nate-double-u suggested, is it possible to test these changes before merging?
Thanks.

**Comment by kbhawkey** [comment]
The changes look okay, though, GitHub issue forms are a beta feature.
/lgtm

**Comment by sftim** [comment]
I tested this as best I could and&hellip; I don't think the feature's ready. GitHub doesn't seem to know if it wants YAML or Markdown, it's not clear what file extension you might use.  I'm going to close this for now.
/lgtm cancel
/close

**Pull request closed without merging**